src/Microsoft.DotNet.XHarness.Android/InstrumentationRunner.cs

```diff
@@ -48,6 +48,7 @@ public InstrumentationRunner(ILogger logger, AdbRunner runner)
 
         bool processCrashed = false;
         bool failurePullingFiles = false;
+        bool? logCatSucceeded = null;
```

Comment:
`DumpAdbLog` returns bool not `Nullable<bool>` and always returns value. I believe it's better to have `bool logCatSucceeded = true`

Reply:
Ah, good point, `DumpAdbLog` is actually called always. I had it originally in a different place. Will fix!